### PR TITLE
Creating Genomic AW5 Raw Table

### PR DESCRIPTION
**Pull request by vmoniqb**
## Description of changes/additions
Creating a table in the RDR to store AW5 Manifest raw data so that it can be included in SPoT.

## Tests
- [X] unit tests

[DA-4417]: https://precisionmedicineinitiative.atlassian.net/browse/DA-4417?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: rdr_service/gcloud_functions/genomic_ingest_manifest_function/main.py [hunk 1]
```diff
@@ -220,7 +220,7 @@ def run(self):
                 "cloud_function": True,
             }
 
-            bypass_raw_keys = ['aw5']
+            bypass_raw_keys = []
```
Comment (j-kanuch) [inline rdr_service/gcloud_functions/genomic_ingest_manifest_function/main.py]: Good catch. Just FYI, Cloud Functions are not deployed during the App Engine deploy, so we need to do this manually.

File: tests/genomics_tests/test_genomic_pipeline.py [hunk 2]
```diff
@@ -881,6 +883,36 @@ def test_ingest_aw5_file(self, metric_cloud_task):
...
+        raw_records = aw5_dao.get_all()
+        raw_records.sort(key=lambda x: x.biobank_id)
+
+        with open_cloud_file(os.path.normpath(f"{bucket_name}/{aw5_subfolder_array}/{test_file_name_aw5_array}")) as csv_file:
+            csv_reader = csv.DictReader(csv_file)
+            file_rows = list(csv_reader)
+
+            # Check rows in file against records in raw table
+            i = 0
+            for file_row in file_rows:
```
Comment (j-kanuch) [inline tests/genomics_tests/test_genomic_pipeline.py]: what about `for i, file_row in enumerate(file_rows):`

File: rdr_service/gcloud_functions/genomic_ingest_manifest_function/main.py (hunk 1 above)
Comment (dtharpeuno) [reply rdr_service/gcloud_functions/genomic_ingest_manifest_function/main.py]: We should just remove this list for bypassing and the accompanying conditional for the check.

**Review by dtharpeuno — APPROVED** [review]

**Review by j-kanuch — APPROVED** [review]

File: tests/genomics_tests/test_genomic_pipeline.py (hunk 2 above)
Comment (vmoniqb) [reply tests/genomics_tests/test_genomic_pipeline.py]: Updated this!

**Pull request merged**